source/lib.cpp

```diff
@@ -93,6 +93,18 @@ void GZ_displayLinkInfo() {
     }
 }
 
+void GZ_displayZombieHoverInfo() {
+    //Generates A and B button presses per second
+    char a_presses_str[8];
+    char b_presses_str[8];
+
+    tww_sprintf(a_presses_str, "A: %d", GZ_getAPressesPerWindow());
+    tww_sprintf(b_presses_str, "B: %d", GZ_getBPressesPerWindow());
+
+    Font::GZ_drawStr(a_presses_str, 454.f, 320.f, 0x00CC00FF, g_dropShadows);
+    Font::GZ_drawStr(b_presses_str, 454.f, 340.f, 0xCC0000FF, g_dropShadows);

```

Comment:
Also, I'm not quite sure where these lines should be drawn on the screen.  Should the left margin line up with Link's Debug Info? The colons? Should it be somewhere else entirely?